website/content/contributors/getting-started.md

```diff
@@ -0,0 +1,111 @@
+The TAG also provides a forum for cross-project collaboration and knowledge-sharing, helping to foster a sense of community and cohesion within the CNCF ecosystem. They work closely with project maintainers, contributors, and other community members to provide guidance and support, and to ensure that all projects within the CNCF are able to thrive and evolve in a collaborative and inclusive manner.
+
+For new contributors, the TAG can be a valuable resource, providing guidance on technical issues, as well as offering advice on how to navigate the CNCF ecosystem and get involved in various projects. 
+
+**[The CNCF Mentoring WG (Working Group)](https://github.com/cncf/mentoring)** is an initiative aimed at providing support and guidance to new contributors looking to get involved in the CNCF community. The group focuses on creating a welcoming and inclusive environment for individuals of all backgrounds and skill levels to contribute to open source projects. One of the main objectives of the group is to identify and promote "good first issues," which are beginner-friendly tasks that can help new contributors gain confidence and familiarity with the project's codebase and development workflow. Be sure to visit the [CLOTributor](https://clotributor.dev/) website to discover great opportunities to become a Cloud Native contributor.
+Additionally, the Mentoring Working Group provides resources such as documentation, tutorials, and one-on-one mentorship to help new contributors navigate the project's technical and social aspects.
+
+**Training material:** The [Linux Foundation](https://training.linuxfoundation.org/) offers a range of free and paid training materials and certifications that can help you upskill and enhance your knowledge in various areas, including cloud computing, DevOps, and open source technologies. 
+These resources include online courses, certifications, and hands-on training programs that are taught by experienced instructors and industry professionals. By taking advantage of these resources, you can gain the skills and knowledge you need to become a valuable contributor to open source projects and communities. 
+### Why open source projects look for new contributors, why should I care to join?
+Open source projects are always in need of new contributors because they rely on the community to continue growing and improving. By joining an open source project, you can not only improve your own skills and knowledge but also contribute to something that has the potential to make a difference in the world. You can also collaborate with like-minded individuals from all over the world, learn from others, and build a network that can benefit you in the future. Additionally, contributing to open source projects is a great way to give back to the community and make a positive impact.
```

Comment:
```suggestion
Open source projects always need new contributors because they rely on the community to continue growing and improving. By joining an open source project, you can not only enhance your skills and knowledge but also contribute to something that has the potential to make a difference in the world. You can also collaborate with like-minded individuals worldwide, learn from others, and build a network that can benefit you in the future. Additionally, contributing to open source projects is a great way to give back to the community and make a positive impact.
```